packages/common/src/prompts/conditions.ts

```diff
@@ -121,6 +21,7 @@ function toString(values: ConditionValue) {
 
 export const conditionOps = {
   eq: ({ answer, value }: ConditionValueOps) => isEqual(toString(answer), toString(value)),
+  setEq: ({ answer, value }: ConditionValueOps) => isEqual(toString(answer), toString(value)),
```

Comment:
This is mostly for clarity because with the current value input system you could get things like "recallNumber = [ "2" ]" or  "recallNumber = [ "2", "3" ]" for a strict equality check which looks confusing and doesn't make sense for a simple value comparison